providers/snowflake/schema_grant.go

```diff
@@ -135,6 +136,8 @@ func (g SchemaGrantGenerator) createResources(schemaGrantList []schemaGrant) ([]
 		switch grant.GrantedTo.String {
 		case "ROLE":
 			tfGrant.Roles = append(tfGrant.Roles, grant.GranteeName.String)
+		case "SHARE":
```

Comment:
Added this because we have share grants as well, and they're not natively supported by the Snowflake provider, so just dropping for now